lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,52 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        tunit = Unit('days since 0001-01-01 00:00:00', calendar='gregorian')
+        tcoord = DimCoord(np.arange(2), standard_name='time', units=tunit)
+        xcoord = DimCoord(np.arange(3), standard_name='longitude',
+                          units='degrees')
+        ycoord = DimCoord(points=np.arange(4))
+        cube.add_dim_coord(tcoord, 0)
+        cube.add_dim_coord(xcoord, 1)
+        cube.add_dim_coord(ycoord, 2)
+        return cube
+
+    def convert_cube_to_field(self, cube):
+        field = iris.fileformats.pp.PPField3()
+        field.lbvc = 0
+        iris.fileformats.pp._ensure_save_rules_loaded()
+        iris.fileformats.pp._save_rules.verify(cube, field)
+        return field
+
+    def test_time_mean_only(self):
+        cube = self.create_cube()
+        cube.cell_methods = (
+            iris.coords.CellMethod(method='mean', coords='time'),
+        )
+        field = self.convert_cube_to_field(cube)
+        self.assertEqual(int(field.lbproc), 128)
+
+    def test_zonal_mean_only(self):
+        cube = self.create_cube()
+        cube.cell_methods = (
+            iris.coords.CellMethod(method=u'mean', coords=u'longitude'),
+        )
+        field = self.convert_cube_to_field(cube)
+        self.assertEqual(int(field.lbproc), 64)
+
+    def test_time_mean_and_zonal_mean(self):
+        cube = self.create_cube()
+        cube.cell_methods = (
+            iris.coords.CellMethod(method=u'mean', coords=u'time'),
+            iris.coords.CellMethod(method=u'mean', coords=u'longitude')
+        )
+        field = self.convert_cube_to_field(cube)
+        self.assertEqual(int(field.lbproc), 192)
+
```

Comment:
@bjlittle I've only added a single test for grid_longitude mean without time mean- I expressed misgivings above about the potential redundancy of grid longitude tests.  That said, I can trivially replicate the longitude mean + time mean test to grid_longitude mean + time mean, if you prefer?